library/src/main/java/com/bumptech/glide/request/target/ImageViewTarget.java

```diff
@@ -45,9 +45,11 @@ public void setDrawable(Drawable drawable) {
      */
     @Override
     public void onLoadStarted(Drawable placeholder) {
+        if (placeholder == null && view.getDrawable() != null) {
```

Comment:
After more thought, we can't make this change. A typical use case where this will occur and you won't want it is in a RecyclerView or ListView. The previous image may have been loaded into the ImageView. If you don't clear it out, even when the placeholder is null, you're going to end up showing the old image while the new one loads, which isn't right.

I'm not sure there's going to be a good way to generalize this, I think you're going to be stuck passing in the placeholder to Glide.

Reply:
@sjudd yeah i think you are right, have to clear it in recyclerview otherwise it will mess up, make sense!
We will just end up using placeholder() and thumbnail() together .
Thanks

Reply:
Thank you, I appreciate you putting this up anyway, it's super helpful to see exactly what you're trying to do.